### ISSUE TITLE
Mod wont start in minecraft

### ISSUE THREAD

**Issue by EnvyElliott** [Globox1997/Dehydration#97]
i'm very new to all of this so if anyone could take the time and really dumb it down for my feeble brain that would be awsome...

here is the error report,

---- Minecraft Crash Report ----
// Who set us up the TNT?

Time: 2022-09-05 08:35:57
Description: Initializing game

java.lang.RuntimeException: Failed to obtain root directory for mod 'dehydration'!
	at net.fabricmc.loader.impl.ModContainerImpl.obtainRootPaths(ModContainerImpl.java:148)
	at net.fabricmc.loader.impl.ModContainerImpl.getRootPaths(ModContainerImpl.java:100)
	at net.fabricmc.fabric.impl.resource.loader.ModNioResourcePack.create(ModNioResourcePack.java:69)
	at net.fabricmc.fabric.impl.resource.loader.ModResourcePackUtil.appendModResourcePacks(ModResourcePackUtil.java:65)
	at net.fabricmc.fabric.impl.resource.loader.ModResourcePackCreator.method_14453(ModResourcePackCreator.java:71)
	at net.fabricmc.fabric.impl.resource.loader.ModResourcePackCreator.register(ModResourcePackCreator.java:52)
	at net.minecraft.class_315.handler$zmg000$onLoad(class_315.java:2217)
	at net.minecraft.class_315.method_1636(class_315.java:1414)
	at net.minecraft.class_315.<init>(class_315.java:1220)
	at net.minecraft.class_310.<init>(class_310.java:461)
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)
Caused by: java.io.IOException: Error accessing file:///C:/Users/ellio/AppData/Roaming/.minecraft/mods/Dehydration-Mod-Fabric-1.19.2.jar: java.util.zip.ZipException: zip END header not found
	at net.fabricmc.loader.impl.util.FileSystemUtil.getJarFileSystem(FileSystemUtil.java:83)
	at net.fabricmc.loader.impl.util.FileSystemUtil.getJarFileSystem(FileSystemUtil.java:59)
	at net.fabricmc.loader.impl.ModContainerImpl.obtainRootPath(ModContainerImpl.java:156)
	at net.fabricmc.loader.impl.ModContainerImpl.obtainRootPaths(ModContainerImpl.java:137)
	... 14 more
Caused by: java.util.zip.ZipException: zip END header not found
	at jdk.zipfs/jdk.nio.zipfs.ZipFileSystem.findEND(ZipFileSystem.java:1320)
	at jdk.zipfs/jdk.nio.zipfs.ZipFileSystem.initCEN(ZipFileSystem.java:1534)
	at jdk.zipfs/jdk.nio.zipfs.ZipFileSystem.<init>(ZipFileSystem.java:179)
	at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.getZipFileSystem(ZipFileSystemProvider.java:125)
	at jdk.zipfs/jdk.nio.zipfs.ZipFileSystemProvider.newFileSystem(ZipFileSystemProvider.java:106)
	at java.base/java.nio.file.FileSystems.newFileSystem(FileSystems.java:339)
	at java.base/java.nio.file.FileSystems.newFileSystem(FileSystems.java:288)
	at net.fabricmc.loader.impl.util.FileSystemUtil.getJarFileSystem(FileSystemUtil.java:78)
	... 17 more

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- Head --
Thread: Render thread
Stacktrace:
	at net.fabricmc.loader.impl.ModContainerImpl.obtainRootPaths(ModContainerImpl.java:148)
	at net.fabricmc.loader.impl.ModContainerImpl.getRootPaths(ModContainerImpl.java:100)
	at net.fabricmc.fabric.impl.resource.loader.ModNioResourcePack.create(ModNioResourcePack.java:69)
	at net.fabricmc.fabric.impl.resource.loader.ModResourcePackUtil.appendModResourcePacks(ModResourcePackUtil.java:65)
	at net.fabricmc.fabric.impl.resource.loader.ModResourcePackCreator.method_14453(ModResourcePackCreator.java:71)
	at net.fabricmc.fabric.impl.resource.loader.ModResourcePackCreator.register(ModResourcePackCreator.java:52)
	at net.minecraft.class_315.handler$zmg000$onLoad(class_315.java:2217)
	at net.minecraft.class_315.method_1636(class_315.java:1414)
	at net.minecraft.class_315.<init>(class_315.java:1220)
	at net.minecraft.class_310.<init>(class_310.java:461)

-- Initialization --
Details:
	Modules: 
		ADVAPI32.dll:Advanced Windows 32 Base API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		COMCTL32.dll:User Experience Controls Library:6.10 (WinBuild.160101.0800):Microsoft Corporation
		CRYPT32.dll:Crypto API32:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTBASE.dll:Base cryptographic API DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		CRYPTSP.dll:Cryptographic Service Provider API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		DBGHELP.DLL:Windows Image Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		DNSAPI.dll:DNS Client API DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		GDI32.dll:GDI Client DLL:10.0.22000.832 (WinBuild.160101.0800):Microsoft Corporation
		IMM32.DLL:Multi-User Windows IMM32 API Client DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		IPHLPAPI.DLL:IP Helper API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		KERNEL32.DLL:Windows NT BASE API Client DLL:10.0.22000.708 (WinBuild.160101.0800):Microsoft Corporation
		KERNELBASE.dll:Windows NT BASE API Client DLL:10.0.22000.708 (WinBuild.160101.0800):Microsoft Corporation
		NSI.dll:NSI User-mode interface DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		NTASN1.dll:Microsoft ASN.1 API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		Ole32.dll:Microsoft OLE for Windows:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		OleAut32.dll:OLEAUT32.DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		PSAPI.DLL:Process Status Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		Pdh.dll:Windows Performance Data Helper DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		PhysXCommon_64.dll:PhysXCommon 64bit Dynamic Link Library:4.1.2.0:NVIDIA Corporation
		PhysXCooking_64.dll:PhysXCooking 64bit Dynamic Link Library:4.1.2.0:NVIDIA Corporation
		PhysXFoundation_64.dll:PhysXFoundation 64bit Dynamic Link Library:4.1.2.0:NVIDIA Corporation
		PhysXJniBindings_64.dll
		PhysX_64.dll:PhysX 64bit Dynamic Link Library:4.1.2.0:NVIDIA Corporation
		RPCRT4.dll:Remote Procedure Call Runtime:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		SHCORE.dll:SHCORE:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		SHELL32.dll:Windows Shell Common Dll:10.0.22000.708 (WinBuild.160101.0800):Microsoft Corporation
		UMPDC.dll:User Mode Power Dependency Coordinator:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		USER32.dll:Multi-User Windows USER API Client DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		USERENV.dll:Userenv:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		VCRUNTIME140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		VERSION.dll:Version Checking and File Installation Libraries:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WINHTTP.dll:Windows HTTP Services:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WINMM.dll:MCI API DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WS2_32.dll:Windows Socket 2.0 32-Bit DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		WSOCK32.dll:Windows Socket 32-Bit DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		amsi.dll:Anti-Malware Scan Interface:10.0.22000.593 (WinBuild.160101.0800):Microsoft Corporation
		bcrypt.dll:Windows Cryptographic Primitives Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		bcryptPrimitives.dll:Windows Cryptographic Primitives Library:10.0.22000.778 (WinBuild.160101.0800):Microsoft Corporation
		clbcatq.dll:COM+ Configuration Catalog:2001.12.10941.16384 (WinBuild.160101.0800):Microsoft Corporation
		combase.dll:Microsoft COM for Windows:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dbgcore.DLL:Windows Core Debugging Helpers:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc.DLL:DHCP Client Service:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		dhcpcsvc6.DLL:DHCPv6 Client:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		eamsi.dll:ESET Antimalware Scan Interface DLL:10.27.46.0 BETA:ESET
		fwpuclnt.dll:FWP/IPsec User-Mode API:10.0.22000.258 (WinBuild.160101.0800):Microsoft Corporation
		gdi32full.dll:GDI Client DLL:10.0.22000.832 (WinBuild.160101.0800):Microsoft Corporation
		glfw.dll:GLFW 3.4.0 DLL:3.4.0:GLFW
		java.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		javaw.exe:OpenJDK Platform binary:17.0.3.0:Microsoft
		jemalloc.dll
		jimage.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jli.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jna4897104571720212823.dll:JNA native library:6.1.2:Java(TM) Native Access (JNA)
		jsvml.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		jvm.dll:OpenJDK 64-Bit server VM:17.0.3.0:Microsoft
		kernel.appcore.dll:AppModel API Host:10.0.22000.71 (WinBuild.160101.0800):Microsoft Corporation
		lwjgl.dll
		management.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		management_ext.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		msvcp140.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		msvcp_win.dll:Microsoft® C Runtime Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		msvcrt.dll:Windows NT CRT DLL:7.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		mswsock.dll:Microsoft Windows Sockets 2.0 Service Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		napinsp.dll:E-mail Naming Shim Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		ncrypt.dll:Windows NCrypt Router:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		net.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		nio.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		nlansp_c.dll:NLA Namespace Service Provider DLL:10.0.22000.653 (WinBuild.160101.0800):Microsoft Corporation
		ntdll.dll:NT Layer DLL:10.0.22000.708 (WinBuild.160101.0800):Microsoft Corporation
		perfos.dll:Windows System Performance Objects DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		pfclient.dll:SysMain Client:10.0.22000.653 (WinBuild.160101.0800):Microsoft Corporation
		pnrpnsp.dll:PNRP Name Space Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		powrprof.dll:Power Profile Helper DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		profapi.dll:User Profile Basic API:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		rasadhlp.dll:Remote Access AutoDial Helper:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		rsaenh.dll:Microsoft Enhanced Cryptographic Provider:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		sechost.dll:Host for SCM/SDDL/LSA Lookup APIs:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		shlwapi.dll:Shell Light-weight Utility Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		sunmscapi.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		ucrtbase.dll:Microsoft® C Runtime Library:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		vcruntime140_1.dll:Microsoft® C Runtime Library:14.29.30139.0 built by: vcwrkspc:Microsoft Corporation
		verify.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
		win32u.dll:Win32u:10.0.22000.856 (WinBuild.160101.0800):Microsoft Corporation
		windows.storage.dll:Microsoft WinRT Storage API:10.0.22000.184 (WinBuild.160101.0800):Microsoft Corporation
		winrnr.dll:LDAP RnR Provider DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		wintypes.dll:Windows Base Types DLL:10.0.22000.1 (WinBuild.160101.0800):Microsoft Corporation
		wshbth.dll:Windows Sockets Helper DLL:10.0.22000.653 (WinBuild.160101.0800):Microsoft Corporation
		zip.dll:OpenJDK Platform binary:17.0.3.0:Microsoft
Stacktrace:
	at net.minecraft.client.main.Main.method_44604(Main.java:205)
	at net.minecraft.client.main.Main.main(Main.java:51)
	at net.fabricmc.loader.impl.game.minecraft.MinecraftGameProvider.launch(MinecraftGameProvider.java:461)
	at net.fabricmc.loader.impl.launch.knot.Knot.launch(Knot.java:74)
	at net.fabricmc.loader.impl.launch.knot.KnotClient.main(KnotClient.java:23)

-- System Details --
Details:
	Minecraft Version: 1.19.2
	Minecraft Version ID: 1.19.2
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 17.0.3, Microsoft
	Java VM Version: OpenJDK 64-Bit Server VM (mixed mode), Microsoft
	Memory: 1113772704 bytes (1062 MiB) / 1577058304 bytes (1504 MiB) up to 2147483648 bytes (2048 MiB)
	CPUs: 24
	Processor Vendor: AuthenticAMD
	Processor Name: AMD Ryzen 9 5900X 12-Core Processor            
	Identifier: AuthenticAMD Family 25 Model 33 Stepping 0
	Microarchitecture: unknown
	Frequency (GHz): 3.69
	Number of physical packages: 1
	Number of physical CPUs: 12
	Number of logical CPUs: 24
	Graphics card #0 name: NVIDIA GeForce RTX 3080 Ti
	Graphics card #0 vendor: NVIDIA (0x10de)
	Graphics card #0 VRAM (MB): 4095.00
	Graphics card #0 deviceId: 0x2208
	Graphics card #0 versionInfo: DriverVersion=31.0.15.1659
	Memory slot #0 capacity (MB): 16384.00
	Memory slot #0 clockSpeed (GHz): 2.67
	Memory slot #0 type: DDR4
	Memory slot #1 capacity (MB): 16384.00
	Memory slot #1 clockSpeed (GHz): 2.67
	Memory slot #1 type: DDR4
	Virtual memory max (MB): 37554.06
	Virtual memory used (MB): 16009.73
	Swap memory total (MB): 4864.00
	Swap memory used (MB): 57.99
	JVM Flags: 9 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xss1M -Xmx2G -XX:+UnlockExperimentalVMOptions -XX:+UseG1GC -XX:G1NewSizePercent=20 -XX:G1ReservePercent=20 -XX:MaxGCPauseMillis=50 -XX:G1HeapRegionSize=32M
	Fabric Mods: 
		architectury: Architectury 6.2.43
		cloth-config: Cloth Config v8 8.1.77
			cloth-basic-math: cloth-basic-math 0.6.1
		dehydration: Dehydration 1.2.10
		fabric-api: Fabric API 0.60.0+1.19.2
			fabric-api-base: Fabric API Base 0.4.11+e62f51a390
			fabric-api-lookup-api-v1: Fabric API Lookup API (v1) 1.6.9+9ff28f4090
			fabric-biome-api-v1: Fabric Biome API (v1) 9.0.17+aeb40ebe90
			fabric-blockrenderlayer-v1: Fabric BlockRenderLayer Registration (v1) 1.1.20+aeb40ebe90
			fabric-client-tags-api-v1: Fabric Client Tags 1.0.1+b35fea8390
			fabric-command-api-v1: Fabric Command API (v1) 1.2.11+f71b366f90
			fabric-command-api-v2: Fabric Command API (v2) 2.1.7+0c17ea9690
			fabric-commands-v0: Fabric Commands (v0) 0.2.28+df3654b390
			fabric-containers-v0: Fabric Containers (v0) 0.1.34+df3654b390
			fabric-content-registries-v0: Fabric Content Registries (v0) 3.2.3+aeb40ebe90
			fabric-convention-tags-v1: Fabric Convention Tags 1.1.1+7cd20a1490
			fabric-crash-report-info-v1: Fabric Crash Report Info (v1) 0.2.5+aeb40ebe90
			fabric-data-generation-api-v1: Fabric Data Generation API (v1) 5.1.8+a680b9b490
			fabric-dimensions-v1: Fabric Dimensions API (v1) 2.1.30+aeb40ebe90
			fabric-entity-events-v1: Fabric Entity Events (v1) 1.4.18+9ff28f4090
			fabric-events-interaction-v0: Fabric Events Interaction (v0) 0.4.28+aeb40ebe90
			fabric-events-lifecycle-v0: Fabric Events Lifecycle (v0) 0.2.28+df3654b390
			fabric-game-rule-api-v1: Fabric Game Rule API (v1) 1.0.21+aeb40ebe90
			fabric-item-api-v1: Fabric Item API (v1) 1.5.7+35a03c4390
			fabric-item-groups-v0: Fabric Item Groups (v0) 0.3.28+aeb40ebe90
			fabric-key-binding-api-v1: Fabric Key Binding API (v1) 1.0.20+aeb40ebe90
			fabric-keybindings-v0: Fabric Key Bindings (v0) 0.2.18+df3654b390
			fabric-lifecycle-events-v1: Fabric Lifecycle Events (v1) 2.1.2+aeb40ebe90
			fabric-loot-api-v2: Fabric Loot API (v2) 1.1.3+83a8659290
			fabric-loot-tables-v1: Fabric Loot Tables (v1) 1.1.6+9e7660c690
			fabric-message-api-v1: Fabric Message API (v1) 5.0.3+176380a290
			fabric-mining-level-api-v1: Fabric Mining Level API (v1) 2.1.14+33fbc73890
			fabric-models-v0: Fabric Models (v0) 0.3.17+aeb40ebe90
			fabric-networking-api-v1: Fabric Networking API (v1) 1.2.4+5eb68ef290
			fabric-networking-v0: Fabric Networking (v0) 0.3.21+df3654b390
			fabric-object-builder-api-v1: Fabric Object Builder API (v1) 4.0.11+aeb40ebe90
			fabric-particles-v1: Fabric Particles (v1) 1.0.10+aeb40ebe90
			fabric-registry-sync-v0: Fabric Registry Sync (v0) 0.9.24+aeb40ebe90
			fabric-renderer-api-v1: Fabric Renderer API (v1) 1.0.10+aeb40ebe90
			fabric-renderer-indigo: Fabric Renderer - Indigo 0.6.12+aeb40ebe90
			fabric-renderer-registries-v1: Fabric Renderer Registries (v1) 3.2.20+df3654b390
			fabric-rendering-data-attachment-v1: Fabric Rendering Data Attachment (v1) 0.3.14+aeb40ebe90
			fabric-rendering-fluids-v1: Fabric Rendering Fluids (v1) 3.0.7+aeb40ebe90
			fabric-rendering-v0: Fabric Rendering (v0) 1.1.22+df3654b390
			fabric-rendering-v1: Fabric Rendering (v1) 1.10.16+aeb40ebe90
			fabric-resource-conditions-api-v1: Fabric Resource Conditions API (v1) 2.0.11+e62f51a390
			fabric-resource-loader-v0: Fabric Resource Loader (v0) 0.6.1+aeb40ebe90
			fabric-screen-api-v1: Fabric Screen API (v1) 1.0.26+f05effaf90
			fabric-screen-handler-api-v1: Fabric Screen Handler API (v1) 1.2.13+9ff28f4090
			fabric-textures-v0: Fabric Textures (v0) 1.0.20+aeb40ebe90
			fabric-transfer-api-v1: Fabric Transfer API (v1) 2.0.11+83a8659290
			fabric-transitive-access-wideners-v1: Fabric Transitive Access Wideners (v1) 1.3.0+42d99c3290
		fabricloader: Fabric Loader 0.14.9
		fallingleaves: Falling Leaves 1.12.2+1.19.2
		java: OpenJDK 64-Bit Server VM 17
		minecraft: Minecraft 1.19.2
		modmenu: Mod Menu 4.0.6
		optifabric: OptiFabric 1.13.16
			mm: Manningham Mills 2.3
		physicsmod: Physics Mod 2.7.6
		sound_physics_remastered: Sound Physics Remastered 1.19.2-1.0.12
	Launched Version: fabric-loader-0.14.9-1.19.2
	Backend library: LWJGL version 3.3.1 SNAPSHOT
	Backend API: Unknown
	Window size: <not initialized>
	GL Caps: Using framebuffer using OpenGL 3.2
	GL debug messages: <disabled>
	Using VBOs: Yes
	Is Modded: Definitely; Client brand changed to 'fabric'
	Type: Client (map_client.txt)
	CPU: <unknown>
	OptiFine Version: OptiFine_1.19.2_HD_U_H9
	OptiFine Build: 20220820-230904
	Shaders: null
	OpenGlVersion: null
	OpenGlRenderer: null
	OpenGlVendor: null
	CpuCount: 0

-- OptiFabric --
Details:
	OptiFine jar designed for: 1.19.2
	OptiFine jar version: OptiFine_1.19.2_HD_U_H9
	OptiFine jar status: Valid OptiFine installer
	OptiFine remapped jar: C:/Users/ellio/AppData/Roaming/.minecraft/.optifine/OptiFine_1.19.2_HD_U_H9/Optifine-mapped.jar
	OptiFabric error: <None>

**Comment by Globox1997** [comment]
I have actually no idea how this happened, do you have the file zipped?

**Comment by Globox1997** [comment]
Reopen with more info and the latest log file.
